Windows compatibility issue

The project's build with Builder Template is failing on Windows during execution `bin\bazel.exe test //...`:

<pre>
[2 / 10] [Prepa] Creating source manifest for //module1:module1
ERROR: D:/a/bazel-ls-demo-project/bazel-ls-demo-project/module1/BUILD:13:1: Building module1/module1-class.jar (1 source file, 1 source jar) failed (Exit 1)
/mybuilder_sample_generation/MybuilderSampleGeneration.java:5: error: illegal unicode escape
    // user.dir: C:\users\runneradmin\_bazel_runneradmin\4udqeaes\execroot\bazel_ls_demo_project
                     ^
/mybuilder_sample_generation/MybuilderSampleGeneration.java:6: error: illegal unicode escape
    //      '.': C:\users\runneradmin\_bazel_runneradmin\4udqeaes\execroot\bazel_ls_demo_project
                     ^
/mybuilder_sample_generation/MybuilderSampleGeneration.java:9: error: illegal unicode escape
    //           C:\users\runneradmin\_bazel_runneradmin\4udqeaes\execroot\bazel_ls_demo_project\bazel-out\x64_windows-fastbuild\bin\module1\mybuilder_generated (normalized)
                     ^
INFO: Elapsed time: 147.018s, Critical Path: 11.79s
//module2:module2-test                                                NO STATUS
INFO: 20 processes: 13 local, 7 worker.
FAILED: Build did NOT complete successfully

Executed 0 out of 1 test: 1 was skipped.
FAILED: Build did NOT complete successfully
</pre>

--- a/src/main/java/com/salesforce/bazel/javabuilder/mybuilder/MyBuilderCommand.java
+++ b/src/main/java/com/salesforce/bazel/javabuilder/mybuilder/MyBuilderCommand.java
@@ -22,6 +22,11 @@ import picocli.CommandLine.Option;
 @Command(name = "mybuilder", description = "A sample builder which generates a file")
 public class MyBuilderCommand implements Callable<Integer> {
 
+    private static String normalizePath(Object input) {
+        // Windows outputs '\' which breaks .java files
+        return String.valueOf(input).replace('\\', '/');
+    }
+
     @Option(names = {"-o", "--output"}, description = "output directory")
     private Path outputDirectory;
 
@@ -41,18 +46,18 @@ public class MyBuilderCommand implements Callable<Integer> {
         lines.add("");
         lines.add(format("public class %s {", className));
         lines.add("");
-        lines.add(format("    // user.dir: %s", System.getProperty("user.dir")));
-        lines.add(format("    //      '.': %s", get(".").toAbsolutePath().normalize()));
+        lines.add(format("    // user.dir: %s", normalizePath(System.getProperty("user.dir"))));
+        lines.add(format("    //      '.': %s", normalizePath(get(".").toAbsolutePath().normalize())));
         lines.add("");
-        lines.add(format("    //   output: %s", outputDirectory));
-        lines.add(format("    //           %s (normalized)", outputDirectory.toAbsolutePath().normalize()));
+        lines.add(format("    //   output: %s", normalizePath(outputDirectory)));
+        lines.add(format("    //           %s (normalized)", normalizePath(outputDirectory.toAbsolutePath().normalize())));
         lines.add("");
         inputFiles.forEach(p -> {
 
             try {
-                lines.add(format("    // %s (%d bytes)", p.toString(), size(p)));
+                lines.add(format("    // %s (%d bytes)", normalizePath(p), size(p)));
             } catch (IOException e) {
-                lines.add(format("    // %s: %s", p.toString(), e.getMessage()));
+                lines.add(format("    // %s: %s", normalizePath(p), e.getMessage()));
             }
 
         });

